components/common/double-tap.tsx

```diff
@@ -0,0 +1,24 @@
+import React, {useRef} from 'react';
+import {State, TapGestureHandler} from 'react-native-gesture-handler';
+
+export const DoubleTap = ({children, customTap}: any) => {
+  const doubleTapRef = useRef(null);
+
+  const onDoubleTapEvent = (event: any) => {
+    if (event.nativeEvent.state === State.ACTIVE) {
+      console.log(customTap);
```

Comment:
Do we need this console.log for any reason?

Reply:
No, we don't. Removed it.